lib/tt_eth/transactions/eip1559_transaction.ex

```diff
@@ -0,0 +1,147 @@
+defmodule TTEth.Transactions.EIP1559Transaction do
```

Comment:
Impressive...

But somehow this looks a bit scary because if something goes wrong, this thing is a bit of an obscure blackbox to debug.
Do you think we should add logs to the Transaction.new (and possibly `Transaction.serialize`) so that we have the value of `trx` and each transformation step in the logs?